config/config-mp/src/test/java/io/helidon/config/mp/MpMetaConfigTest.java

```diff
@@ -57,7 +60,7 @@ void resetConfig() {
         }
     }
 
-    @Test
+    // @Test
```

Comment:
This test should be removed (as yaml is now not a part of dependencies of this module). Equivalent test should exist somewhere else.

Reply:
Removing